src/mpl2/src/block_placement.cpp

```diff
@@ -954,7 +954,7 @@ void SimulatedAnnealingCore::CalculateNotchPenalty()
   AlignMacro();
   vector<float> x_vec;
   vector<float> y_vec;
-  for(int i = 0; i < blocks_.size(); i++)
+  for(int i = 0; i < blocks_.size(); i++) {
```

Comment:
could be `for(const auto& block : blocks_)`

Reply:
Solved